QUEST.tsv

```diff
@@ -45,251 +45,251 @@ QUEST_20150317_000044	Items to be deleted
-QUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want first.
+QUEST_20150317_000147	Clerics work with Intelligence, but might need a lot of Strength or Spirit. {nl}You should really decide what you want, first.
 QUEST_20150317_000148	Doppelsoeldner Master
-QUEST_20150317_000149	The things that are worthwhile will always carry a degree of risk. {nl}Naturally, for big rewards, you'll have to put up with danger.
+QUEST_20150317_000149	Worthwhile things always carry a degree of risk. {nl}Naturally, for the big rewards, you'll have to put up with lots of danger.
 QUEST_20150317_000150	Corsair Master
-QUEST_20150317_000151	I won't help anyone in trouble if there isn't anything in it for me.
+QUEST_20150317_000151	If there isn't anything in it for me, I won't lift a finger.
 QUEST_20150317_000152	Boulder
 QUEST_20150317_000153	A large boulder is blocking the way.
 QUEST_20150317_000154	It needs to be repaired.
 QUEST_20150317_000155	A boulder blocks the way.
 QUEST_20150317_000156	Magical Research Material
-QUEST_20150317_000157	It appears to be someone's research notes from the Mage Tower. {nl}Should it be copied?
-QUEST_20150317_000158	Not everything can be done with hiding and sneaking. {nl}A true Scout must also be ready for a fight.
-QUEST_20150317_000159	You'll need a lot of little things while you're travelling. {nl}You should buy up your provisions in advance.
+QUEST_20150317_000157	It appears to be someone's research notes from Mage Tower. {nl}Should it be copied?
+QUEST_20150317_000158	Hiding and sneaking won't solve all your problems. {nl}A true Scout must also be ready to fight.
+QUEST_20150317_000159	You're gonna need a lot of things when travelling. {nl}Be sure to stock up your provisions in advance.
 QUEST_20150317_000160	If the control magic is dispelled, the divine messages won't be protected.
 QUEST_20150317_000161	Forgotten Thief's Cenotaph
-QUEST_20150317_000162	I, Zubeck, leave this inscription. {nl}For the honor of being the greatest thief in history, I challenged a powerful sorcerer.
-QUEST_20150317_000163	But could I, a mere thief, claim victory? {nl}Naturally, I would know by cleaning up an opponent like a great wizard.
-QUEST_20150317_000164	Yet, I couldn't. {nl}I wanted to be the number one thief of all time, not just of today. 
-QUEST_20150317_000165	I have at last discovered a way to enter the Mage Tower. {nl}The power must be gathered into a special location - it's only a matter of time.
-QUEST_20150317_000166	Then, my special device will activate. {nl}If that is repeated, you might also be able to enter the Mage Tower.
-QUEST_20150317_000167	Obviously, when you're done, you can show those dumb mages a thing or two. Haha!
+QUEST_20150317_000162	I, Zubeck, leave this epitaph. {nl}For the pride of being the greatest thief in history, I challenged a powerful sorcerer.
```

Comment:
I believe "For the pride of" is incorrect, in favor of the original "For the honor of".  If you wanted to use the word "pride" specifically, something like "To gain the pride of" would be correct.